packages/cardano-services/src/PgBoss/stakePoolHandler.ts

```diff
@@ -0,0 +1,68 @@
+import { Cardano } from '@cardano-sdk/core';
+import { DataSource } from 'typeorm';
+import { Hash32ByteBase16 } from '@cardano-sdk/crypto';
+import { Observable, firstValueFrom } from 'rxjs';
+import { PoolMetadataEntity, StakePoolMetadataTask } from '@cardano-sdk/projection-typeorm';
+import { WorkerHandlerFactory } from './types';
+import { createHttpStakePoolMetadataService } from '../StakePool/HttpStakePoolMetadata/HttpStakePoolMetadataService';
```

Comment:
Seems like we're missing a couple of index re-exports
```suggestion
import { createHttpStakePoolMetadataService } from '../StakePool';
```

Reply:
8b9948b40 should solve this